src/yunohost/settings.py

```diff
@@ -46,6 +46,10 @@
         "choices": ["intermediate", "modern"]}),
     ("security.postfix.compatibility", {"type": "enum", "default": "intermediate",
         "choices": ["intermediate", "modern"]}),
+    ("smtp.relay.host", {"type": "string", "default": ""}),
+    ("smtp.relay.port", {"type": "int", "default": 587}),
+    ("smtp.relay.user", {"type": "string", "default": ""}),
+    ("smtp.relay.password", {"type": "string", "default": ""}),
```

Comment:
I believe there are corresponding strings to be defined in `locales/en.json` for these.

Also maybe we want to rename this as `email.relay.host` (and so on) to anticipate the integration in the webadmin where we'll probably want something more friendly and general like "Email" category rather than the technical and specific "SMTP" acronym ?

Reply:
I write some description. 

We have a smtp.allow_ipv6 in 3.8. What should we do ?

Reply:
So after discussion it seems we keep smtp